index.html

```diff
@@ -118,8 +118,7 @@ <h1 id="title">
               Start date
             </th>
             <td>
-              <i class="todo">[dd monthname yyyy] (date of the "Call for
-              Participation", when the charter is approved)</i>
+              <i>1 May 2021</i>
```

Comment:
I think we need to leave this as a TODO since we can't guess when the actual call for participation will be sent.

Reply:
Yes, it makes sense. You can ignore merging the TODO marked above.